.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "main",
```

Comment:
how `baseBranch` relates to release branches here?

Reply:
according to [docs](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#basebranch-git-branch-name) it should be set to `main` as this is used for e.g `changesets --since` :
> This should generally set to the major branch you merge changes into